### PR TITLE
support for now() helper function in SQL Queries

### PR DESCRIPTION
**Pull request by lampersky**
- support for argumentless functions (**SqlParser** & **SqlGrammar**),
- documentation update,
- YesSql version updated to the latest.

fixes https://github.com/OrchardCMS/OrchardCore/issues/8538

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlParser.cs [hunk 1]
```diff
@@ -446,7 +446,9 @@ private void EvaluateFunCall(ParseTreeNode funCall)
             IList<string> arguments;
             var tempBuilder = _builder;
 
-            if (funCall.ChildNodes[1].ChildNodes[0].Term.Name == "selectStatement")
+            if (funCall.ChildNodes[1].ChildNodes.Count == 0) {
+                arguments = new string[] { };
```
Comment (deanmarcussen) [inline src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlParser.cs:450]:
```suggestion
                arguments = Array.Empty<string>();
```

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlParser.cs:450]: and formatting here (drop the braces and else if onto new lines). Thanks

**Comment by deanmarcussen** [comment]
@sebastienros needs to merge dev -> master to get the new YesSql version on nuget :)

**Comment by deanmarcussen** [comment]
YesSql updated now on a different pr. so this pr can be updated (just remove the changes to `Dependencies.props`)

**Comment by lampersky** [comment]
> YesSql updated now on a different pr. so this pr can be updated (just remove the changes to `Dependencies.props`)

Thanks for info @deanmarcussen, will update PR later today

**Review by deanmarcussen — APPROVED** [review]

**Pull request merged**